pkg/util/status/build_status.go

```diff
@@ -70,11 +70,20 @@ const (
 	ReasonMessageInvalidArtifactsMapping api.StepFailureMessage = "Invalid artifacts mapping specified."
 
 	// ReasonArtifactsFetchFailed is the reason associated with a failure to
-	// download specified scripts in the application image.
-	ReasonArtifactsFetchFailed api.StepFailureReason = "FetchScriptsFailed"
+	// download specified artifacts in the application image.
+	ReasonArtifactsFetchFailed api.StepFailureReason = "FetchArtifactsFailed"
 	// ReasonMessageArtifactsFetchFailed is the message associated with a failure to
-	// download specified scripts in the application image.
-	ReasonMessageArtifactsFetchFailed api.StepFailureMessage = "Failed to fetch scripts specified scripts."
+	// download specified artifacts in the application image.
+	ReasonMessageArtifactsFetchFailed api.StepFailureMessage = "Failed to fetch specified artifacts."
```

Comment:
things artifacts could be referring to, in my mind:  

1) remote assemble/run/save-artifacts scripts we were trying to download from an http server
2) running save-artifacts to extract dependencies from the previous image, failed